app/templates/views/features.html

```diff
@@ -10,31 +10,31 @@
 
 <h1 class="heading-large">Features</h1>
 <p class="govuk-body">If you work for the federal government, you can use US Notify to keep your users updated.</p>
-<p class="govuk-body">Notify makes it easy to create, customize and send:</p>
-<ul class="list list-bullet">
+<p class="govuk-body">Notify makes it easy to create, customize, and send <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_sms') }}">text messages</a>.</p>
+<!-- <ul class="list list-bullet">
   <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_email') }}">emails</a></li>
   <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_sms') }}">text messages</a></li>
-  <!-- <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_letters') }}">letters</a></li> -->
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_letters') }}">letters</a></li> -->
 </ul>
```

Comment:
[suggestion] this `</ul>` needs to be part of the comment out, because the matching opening `<ul>` is commented out.
```suggestion
  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_letters') }}">letters</a></li>
</ul> -->
```

Reply:
thanks for that catch!